frontend/src/components/NavBar.js

```diff
@@ -0,0 +1,125 @@
+import { useState } from "react";
+import Svg from "../assets/svg/Svg";
+
+const NavBar = ({ runInfo }) => {
+  const generateDialectNavigation = true;
+  const body = document.querySelector("body");
+  const [mode, setMode] = useState(
+    window.matchMedia("(prefers-color-scheme: dark)").matches,
+  );
+
+  body.setAttribute("data-bs-theme", mode ? "dark" : "light");
+
+  function toggleMode() {
+    const newMode = !mode;
+    setMode(newMode);
+  }
+
+  return (
+    <>
+      <nav
+        className={`navbar navbar-expand-lg sticky-top mb-4 ${
+          mode ? "navbar-dark bg-dark" : "navbar-light bg-light"
+        }`}
+      >
+        <div className="container-fluid">
+          <a className="navbar-brand mb-0 h1" href="#">
+            Bowtie
+          </a>
+          <button
+            className="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNav"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span className="navbar-toggler-icon"></span>
+          </button>
+          <div className="collapse navbar-collapse" id="navbarNav">
+            <ul className="navbar-nav me-auto mb-2 mb-lg-0">
+              <li className="nav-item">
+                <a className="nav-link" href="#run-info">
+                  Run Info
+                </a>
+              </li>
+              <li className="nav-item">
+                <a className="nav-link" href="#summary">
+                  Summary
+                </a>
+              </li>
+              <li className="nav-item">
+                <a className="nav-link" href="#cases">
+                  Details
+                </a>
+              </li>
+              {generateDialectNavigation && (
```

Comment:
This is more of a TODO than something to address now, but now that we have a dynamic site, we should likely also have a dynamic way to populate this navigation -- specifically rather than hardcoding which dialects we support, we should probably simply automatically populate this list by querying which dialects are available. Perhaps just file an issue for us to do so after this PR is merged?

Reply:
Sure, I'll file an issue after this PR gets merged.